build-config.yaml

```diff
@@ -0,0 +1,35 @@
+apps:
+  ckan: &app_ckan
+    name: ckan
+    version: "2.10.4"
+    patch: e
+  pycsw: &app_pycsw
+    name: pycsw
+    version: "2.6.1"
+    patch: k
+  solr: &app_solr
+    name: solr
+    version: "2.10"
+    patch: a
```

Comment:
This will probably need bumping up to `b` so that we can also build it using the same build process and for testing the process now.